src/lib/ProductCard/productCard.module.scss

```diff
@@ -0,0 +1,151 @@
+@import '../variables';
+
+.product-card-container {
+  border: 1px solid #f1f1f2;
+  width: 380px;
+
+  @media screen and (max-width: $tablet) {
+    width: 315px;
+  }
+}
+
+.price-container {
+  padding-top: 11px;
+  padding-left: 10px;
+  display: flex;
+}
+
+.product-image {
+  position: relative;
+
+  img {
+    width: 100%;
+  }
+
+  @media screen and (max-width: $mobile) {
+    img {
+      width: 151px;
+      height: 124px;
+    }
+  }
+}
+
+.match-banner {
+  position: absolute;
+  background-color: $gray-11;
+  color: #f4f4f4;
```

Comment:
for consistency sake - is there anything close to this color on the variables list? maybe gray-2?

Reply:
Hmm.. maybe I can use `$gray-2/$gray-3` but I"m just worried that since it's such a specific color that QA will demolish it as soon as they see it. But that's a good point and it is our component library. Maybe @craig1123 will have a good idea about what to do here because I do agree with you.

Reply:
i mean eventually (ideally) the designs would be consistent in the colors they use.

Reply:
I have a confession to make. $gray-2, $gray-6, $gray-8, $gray-9, $gray-10, and $gray-12 are not used any where and I made up the color. oops. So you can replace whatever color is in gray-2 with that one